eiffel/src/main/java/com/etiennelenhart/eiffel/Eiffel.kt

```diff
@@ -0,0 +1,30 @@
+package com.etiennelenhart.eiffel
+
+import com.etiennelenhart.eiffel.logger.Logger
+
+object Eiffel {
+
+    internal var debugConfig: DebugConfig = DebugConfig.Default
+        private set
+
+    /**
+     * Enable or disable Eiffel's debug mode.
+     *
+     * You can override the [Logger.Default] by passing in your own implementation of [Logger].
+     * @sample
```

Comment:
I just learned that `@sample` is used for referencing file-private sample functions. So for inline samples like this one it would be better to replace `@sample` with `Example:`

Reply:
Well ain't that just annoying.  From the name it sounded perfect 🥇